e2e/src/components/Footer.ts

```diff
@@ -0,0 +1,37 @@
+import { Page, Locator, expect } from '@playwright/test';
+import BaseComponent from '../base/BaseComponent';
+
+export default class Footer extends BaseComponent {
+    private readonly footerLocator = this.page.locator('footer#footer');
+    private readonly linkedInLocator = (this.page.locator('a[href*="linkedin.com"]')).last();
```

Comment:
is it possible to search the locator inside the footer HTML elem? To avoid .last() etc.